pkg/tenant/members.go

```diff
@@ -111,29 +112,55 @@ func (s *Server) MemberCreate(c *gin.Context) {
 		return
 	}
 
-	// Verify that a member name exists and return a 400 response if it does not.
-	if member.Name == "" {
-		c.JSON(http.StatusBadRequest, api.ErrorResponse("member name is required"))
-		return
-	}
```

Comment:
I'm pretty sure we decided to make the member name optional in the database right? At least that's what is represented in Figma currently.

Reply:
I believe that's true yes - or at least the member should be able to update their own name. Would be good to check in on this during the UI sync on Thursday though.